core/src/i18n/en.yaml

```
@@ -146,6 +146,25 @@ applications:
     provisioning: The Application is being provisioned asynchronously.
 apps:
   title: Apps
+authorizationpolicies:
```

we separate words with "-" as a standard 
```suggestion
authorization-policies:
```